GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/UpdatePlaceholderTests.cs

```diff
@@ -34,18 +34,18 @@ public virtual void SetupForTest()
             this.GitStatusShouldBeClean(NewFilesAndChangesCommitId);
         }
 
+        // WindowsOnly because test requires Windows specific file sharing behavior
         [TestCase, Order(1)]

```

Comment:
How important is the relative/interleaved ordering of all the tests in this class? The code would definitely be simpler if we pull all the WindowsOnly tests out into a separate class.

Reply:
I don't think the ordering is that important, and I like the idea.  

I will see if I can pull all of the WindowsOnly tests out into a separate class that lives in the Windows specific functional test project (that also gives us the option of keeping the code that uses `CreateFile` as well).  

Thanks!